app/services/report_importer.rb

```diff
@@ -64,9 +64,10 @@ def name
   end
 
   def host
-    @host ||= Host::Base.find_by_certname(name) ||
-      Host::Base.find_by_name(name) ||
-      Host::Managed.new(:name => name)
+    hostname = name.downcase
+    @host ||= Host::Base.find_by_certname(hostname) ||
+      Host::Base.find_by_name(hostname) ||
```

Comment:
If this is because of Rails 4, it's fine. `find_by_` is not deprecated, all other finders are.